thunder/clang/__init__.py

```diff
@@ -2173,6 +2173,46 @@ def topk(
     return prims.topk(a, k, dim, bool(largest), bool(sorted), out=out)
 
 
+@clangop()
```

Comment:
I would just define these as torch operators;  clang operators are a set of operations that are useful for building PyTorch (or NumPy) operations

Reply:
I see your point. Quite informative, thank you!